Remotion/Web/UnitTests/Core/Services/WebServiceFactoryTests/CreateScriptService.cs

```diff
@@ -42,14 +42,22 @@ private interface IValidScriptService
       void MethodWithParameters (int param1, bool param2);
     }
 
+    private interface IValidInheritedScriptService : IValidScriptService
+    {
```

Comment:
I know that from a whitebox approach, this is probably sufficient. I think a version with methods and one without methods would still be helpful to provide sufficient coverage in the event of...odd changes to the code.